Wkhtmltopdf.php

```diff
@@ -770,6 +771,11 @@ protected function _render()
             throw new Exception("Shell error, return code: " . (int)$content['return']);
         }
 
+        // Clean up temp files
+        $filepath = $this->getFilePath();
```

Comment:
maybe better add this to separate method (`private function cleanup`) & make cleanup in `render` method.